src/go/pt-galera-log-explainer/whois.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+/*
+type whois struct {
+	Search string   `arg:"" name:"search" help:"the identifier (node name, ip, uuid, hash) to search"`
```

Comment:
Also, do we need this file to be commited?

Reply:
Seen with Sveta as well, can keep
Will be re-added later properly, the latest commits commented it out.